.github/workflows/package-and-deploy.yml

```diff
@@ -0,0 +1,112 @@
+name: Package and deploy everest-admin-panel
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        description: Image tag
+        default: 'latest'
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+      - uses: pnpm/action-setup@v2
+        name: Install pnpm
+        with:
+          version: 8
+          run_install: false
+      - name: Get pnpm store directory
+        shell: bash
+        run: |
+          echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+      - uses: actions/cache@v3
+        name: Setup pnpm cache
+        with:
+          path: ${{ env.STORE_PATH }}
+          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-
+      - name: Install dependencies
+        run: pnpm install
+      - name: Build
+        run: pnpm build
+      - name: Tar
+        run: tar -C dist -czf everest-admin-panel.tar.gz ./
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/') && success()
+        with:
+          tag_name: ${{ inputs.tag }}
```

Comment:
The tag name should now come from git instead of a pipeline input. Just looked up how I did that in another pipeline where I use the same action and it turns out you just leave the variable unset entirely. It will then automatically use the correct tag

Reply:
@nzbr can you just copy it over from your other pipeline?